backend/pom.xml

```diff
@@ -77,6 +77,11 @@
             <artifactId>bcprov-jdk15on</artifactId>
             <version>1.70</version>
         </dependency>
+        <dependency>
+            <groupId>org.jacoco</groupId>
+            <artifactId>jacoco-maven-plugin</artifactId>
+            <version>0.8.10</version>
+        </dependency>
```

Comment:
Nicht nötig, da in Plugins bereits konfiguriert

Reply:
Umgesetzt